pkgs/development/python-modules/pyopencl/default.nix

```diff
@@ -12,43 +12,39 @@
   nanobind,
 
   # dependencies
-  appdirs,
-  cffi,
   darwin,
-  decorator,
-  mako,
   numpy,
   ocl-icd,
-  oldest-supported-numpy,
   opencl-headers,
   platformdirs,
   pybind11,
-  pytestCheckHook,
   pytools,
-  six,
+
+  # tests
+  pytestCheckHook,
 }:
 
 let
   os-specific-buildInputs = if stdenv.isDarwin then [ darwin.apple_sdk.frameworks.OpenCL ] else [ ocl-icd ];
 in
 buildPythonPackage rec {
   pname = "pyopencl";
-  version = "2024.2.6";
-  format = "pyproject";
+  version = "2024.2.7";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "inducer";
     repo = "pyopencl";
     rev = "refs/tags/v${version}";
-    hash = "sha256-nP7ZAGeRXrjqDRWlc2SDP1hk1fseGeu9Zx0lOp9Pchs=";
+    fetchSubmodules = true;
+    hash = "sha256-VeaEDYnGfMYf9/WqMIZ9g4KounD48eWF3Romt79RMEQ=";
   };
 
-  nativeBuildInputs = [
+  build-system = [
     cmake
     nanobind
     ninja
     numpy
-    oldest-supported-numpy
     pathspec
```

Comment:
```suggestion
```

with https://github.com/NixOS/nixpkgs/pull/336897

Reply:
I'll rebase once your PR is merged.